disassemblers/ofrak_capstone/ofrak_capstone/components.py

```diff
@@ -89,8 +89,8 @@ async def analyze(self, resource: Resource, config=None) -> Instruction:
         parent_block = await resource.get_parent_as_view(Addressable)
         mode: InstructionSetMode
         if parent_block.resource.has_tag(BasicBlock):
-            bb_attrs = await parent_block.resource.analyze_attributes(BasicBlock.attributes_type)
-            mode = bb_attrs.mode
+            bb_attrs = await parent_block.resource.analyze(BasicBlock.attributes_type)
+            mode = bb_attrs.mode  # type: ignore
```

Comment:
A `typing.cast` on line 92 would probably be clearer than a type ignore below.

Reply:
The type it would cast to is `BasicBlock.attributes_type` which MyPy doesn't understand (its created dynamically). All it can know is that it is `ResourceAttributes` which doesn't have a `mode` attribute.